### ISSUE TITLE
Beacon data saving and establishing proper way to develop Homemade RUM application

### ISSUE THREAD

**Issue by ceckoslab** [revampix/homemade_rum#1]
Currently in folder "proof of concpet/push" we have PHP code that is a proof of concept. This code needs to be changed and ported PHP framework that makes sense. Even if framework is not used probably we will need some DB wrapper in order to avoid writing raw SQL queries and avoid SQL injection.

**For now we may cover 2 points:**
 * Add DB wrapper.
 * Add configuration for DB credentials in order to avoid hardcoding the credentials directly in the code.

**What is needed for local setup:**
 * Database should be created from "proof of concpet/db_schema.sql".
 * File "proof of concpet/rum_js/boom-compiled-navt-rest-guid-mobile.js" should be included in HTML page in order to send data from the browser to the php script that handles beacon data.
 * After we add "boom-compiled-navt-rest-guid-mobile.js" we need to initialise where the beacon data should be send. We can do this by adding in script tag ```BOOMR.init({beacon_url:"url-to-write.php"});```

**This is basic diagram how beacon saving and sending process works:**

<img width="951" alt="beacon save" src="https://user-images.githubusercontent.com/1024001/31466322-1000577e-aed7-11e7-91aa-e0a642792bfa.png">

**Comment by ceckoslab** [comment]
I also added test file which idea is to shorten the development cycle when beacon part is being developed.

This is the file that simulates posting data: https://github.com/revampix/homemade_rum/blob/master/proof%20of%20concpet/test/test-push.php

I also had to add a cookie with name GUID but I also had some issues with this cookie when I was saving it in DB. Seems that the cookie value contains " surrounding the actual value and if the quotes are not there then insert query doesn't work. I guess that best is if we trim the " from cookie value and make the insert work correctly.
